MBBSEmu/Database/Repositories/RepositoryBase.cs

```diff
@@ -14,32 +14,24 @@ namespace MBBSEmu.Database.Repositories
     ///
     ///     Holds common Dapper Functionality for executing Queries
     /// </summary>
-    public abstract class RepositoryBase : IRepositoryBase, IDisposable
+    public abstract class RepositoryBase(ISessionBuilder sessionBuilder, IResourceManager resourceManager) : IRepositoryBase, IDisposable
     {
```

Comment:
is this some fancy C# way to define fields in the constructor rather than inside the class itself?